### DISCUSSION TITLE
How to use pretix for multi day event, that only can booked completely

### DISCUSSION THREAD

**Discussion by AboveNBeyondCode** [pretix/pretix#4167] [Q&A]
Hi,

I tried to solve the following problem:

Event (more like a course) is a set of 5 event days (e.g. 5 Mondays in a row from 6pm to 7pm) that only can booked completely. This means the client only has to pay one ticket and can participate all 5 times. It should not be possible to book single dates. 
After the first event day passed there should be a new ticket option available, that will allow you to buy the last 4 event days for a lower price.

Using the single event feature, the client will see a wrong  iCal / calendar entry for this event.

Using the series feature, the client will see a booking button behind every single day, which may confuse him and let him think he has to book every single day.

Is there a possibility to create a multi event with only one booking option?

greets
Achim

**Comment by raphaelm** [comment]
The best solution is probably a single event and accept the iCal issue, we don't have a proper solution for this yet.
